### ISSUE TITLE
How to add chat colours for admin and chaning name with /nick fix and how to get a coloured name

### ISSUE THREAD

**Issue by sanderDijkxhoorn** [GrowtopiaNoobs/GrowtopiaServer#334]

**Comment by ilyn1337** [comment]
Colors same as normal chat message, isn't it?
/nick \`4MyNick\`2IsSoCool

To fix no nick glitch for example your nick is "Nick1233" and you want to change it to "LOL", and it'll probably disappear, to fix this: do, "/nick LOL`````", it's manual, you can write it to code by adding `nickname + "``````"`

**Comment by sanderDijkxhoorn** [comment]
> Colors same as normal chat message, isn't it?
> /nick `4MyNick`2IsSoCool
> 
> To fix no nick glitch for example your nick is "Nick1233" and you want to change it to "LOL", and it'll probably disappear, to fix this: do, "/nick LOL`````", it's manual, you can write it to code by adding `` nickname + "``````" ``

but the problem is for some reason I can't type ` in GrowTopia I think this is my problem.... I check if I am using my razor keyboard I still can't thanks for your help.

**Comment by sanderDijkxhoorn** [comment]
so if I make it for admin only its like 
if (!canSB(((PlayerInfo*)(peer->data))->rawName, ((PlayerInfo*)(peer->data))->tankIDPass)) continue;

**Comment by ilyn1337** [comment]
Yes, it's your keyboard problem, also yes, you can use isSuperAdmin or isAdmin (I don't know if it's exists)

**Comment by MacaylaMarvelous81** [comment]
idk about chat color but for name when logged in add if statement to check if admin and after setting name to normal make it set to "`#" + name. example:
btw code is not tested but i have similar in mine
```
if(isSuperAdmin())
{
     ((PlayerInfo*)(event.peer->data))->displayName = "`#" + ((PlayerInfo*)(event.peer->data))->tankIDName;
}
```

**Comment by sanderDijkxhoorn** [comment]
Thanks for trying to help me but i accidentally deleted all my files of my pc so I need to remake everything again... and I don’t like to make it again... i added gem system added extra commands fixed things made a item shop so I am a bit sad...

**Comment by ritzysw** [comment]
that's why you must have backup

**Comment by sanderDijkxhoorn** [comment]
Yup....
